packages/kotti-ui/package.json

```diff
@@ -98,5 +98,5 @@
 	"style": "./dist/style.css",
 	"type": "module",
 	"types": "./dist/index.d.ts",
-	"version": "8.3.1"
+	"version": "8.4.0"
```

Comment:
This version is the Kotti-UI version, you don't need to update this unless you want a new Kotti version.

What is still missing, is the Yoco dependency here (in this file). It should match the new yoco version, so: `"@3yourmind/yoco": "^2.10.0"`

Reply:
I felt inclined to take over and do that in @RueRivoli's stead.